lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -0,0 +1,70 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "sorbet-runtime"
```

Comment:
Since this is a separate process, we need to add [this block](https://github.com/Shopify/ruby-lsp/blob/main/exe/ruby-lsp#L73-L86) to turn off runtime checks.